### PR TITLE
KAFKA-14925: Hosting JS files locally

### PR DESCRIPTION
**Pull request by atu-sharm**
Adding one JS file which is recommended to be hosted along with the website so that we don't have a dependency on external resources

### CODE REVIEW

**Comment by atu-sharm** [comment]
@mimaison can you review please

File: includes/_header.htm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		<meta property="og:type" content="website" />
 		<link href="/css/fonts.css" rel="stylesheet">
 		<script src="/js/jquery.min.js"></script>
-		<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
+		<script defer src="/js/all.js"></script>
```
Comment (machi1990) [inline includes/_header.htm]: let's give this file a descriptive name e.g `fontawesome.js`

Comment (atu-sharm) [reply includes/_header.htm]: done

**Comment by atu-sharm** [comment]
@mimaison @machi1990 do i need to add something more?

File: includes/_header.htm (hunk 1 above)
Comment (machi1990) [reply includes/_header.htm]: thanks; The change looks good to me.
let's wait to get a review from the committer as well /cc @mimaison

**Review by machi1990 — APPROVED** [review]

**Review by mimaison — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**